### PR TITLE
Build updates/more int -> long type changes in libnd4j

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?
more int -> sd::LongType changes
Spacing/comment cleanup

## How was this patch tested?

Ran tests.

## Quick checklist

The following checklist helps ensure your PR is complete:

- [X ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.konduit.ai/multi-project/how-to-guides/contribute/eclipse-contributors) page for details
- [X ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ X] Created tests for any significant new code additions.
- [ X] Relevant tests for your changes are passing.

### CODE REVIEW

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp [hunk 1]
```diff
@@ -612,7 +614,7 @@ sd::Status sd::ops::DeclarableOp::validateDataTypes(Context &block) {
           if (_descriptor->isSameMode()) {
             if (index >= block.width()) {
               if (block.width() == 0) continue;
-
+              sd_printf("Before variable validate data types\n",0);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/DeclarableOp.cpp]: Left over from debugging?

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp [hunk 2]
```diff
@@ -622,6 +624,8 @@ sd::Status sd::ops::DeclarableOp::validateDataTypes(Context &block) {
                 return sd::Status::BAD_ARGUMENTS;
               }
             } else {
+              sd_printf("Before variable validate data types 2\n",0);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/DeclarableOp.cpp]: Left over from debugging?

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp [hunk 3]
```diff
@@ -818,23 +822,32 @@ sd::Status sd::ops::DeclarableOp::execute(Context *block) {
 
     for (int e = 0; e < numOutputs; e++) {
       // if given output index doesn't exist - we're done
+      sd_printf("Declarable op execute: processing output %d\n",e);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/DeclarableOp.cpp:821]: Left over from debugging?

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp [hunk 4]
```diff
@@ -1017,6 +1029,7 @@ sd::Status sd::ops::DeclarableOp::validateNonEmptyInput(Context &block) {
 
 sd::Status sd::ops::DeclarableOp::validateOrdersMatch(Context &block) {
   if (block.width() == 0) return sd::Status::OK;
+  sd_printf("Before validateOrdersMatch\n",0);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/DeclarableOp.cpp]: Left over from debugging?

File: libnd4j/include/ops/declarable/impl/PlatformHelper.cpp [hunk 5]
```diff
@@ -56,6 +56,7 @@ sd::NDArray* PlatformHelper::getZ(graph::Context& ctx, int inputId) {
     std::pair<int, int> pair(ctx.nodeId(), inputId);
 
     if (ctx.isInplace()) {
+      sd_printf("Before platform helper getZ validate and execute\n",0);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/PlatformHelper.cpp]: Left over from debugging?

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp [hunk 6]
```diff
@@ -116,6 +116,7 @@ sd::NDArray *sd::ops::DeclarableOp::getZ(Context &ctx, int inputId) {
     std::pair<int, int> pair(ctx.nodeId(), inputId);
 
     if (ctx.isInplace()) {
+      sd_printf("DeclarableOp::getZ(Context &ctx, int inputId) Before ctx.variable in getZ() \n", inputId);
```
Comment (treo) [inline libnd4j/include/ops/declarable/impl/DeclarableOp.cpp]: Left over from debugging?

**Review by treo — CHANGES_REQUESTED** [review]

File: libnd4j/include/ops/declarable/impl/DeclarableOp.cpp (hunk 3 above)
Comment (treo) [reply libnd4j/include/ops/declarable/impl/DeclarableOp.cpp:821]: This one is still around.

Comment (agibsonccc) [reply libnd4j/include/ops/declarable/impl/DeclarableOp.cpp:821]: This one is in the verbose/debug path.

**Review by treo — APPROVED** [review]

**Pull request merged**